src/components/ChatEntry.js

```diff
@@ -1,22 +1,42 @@
 import React from 'react';
+import { useState } from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+
+  const [color, setColor] = useState(false);
+
+  const colorClick = () => {
+    if (color === true) {
+      setColor(false)
+      props.takeLikes()
+    } else if (color === false) {
+      setColor(true)
+      props.addLikes()
+    }
+  }
+  const colorChange = color ? '❤️' : '🤍';
```

Comment:
This would then be refactored to use the `props.liked` value to assign the appropriate heart